docs/product-installation-guide/src/main/asciidoc/clustering-dm-install-proc.adoc

```diff
@@ -33,7 +33,7 @@ Complete the steps in this section to install and run {CENTRAL} on each node of
   <property name="org.appformer.ext.metadata.elastic.retries" value="10"/>
```

Comment:
Not shown, but around here, maybe use "large volumes" instead of "big volumes". Just sounds kind of informal.